src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala

```diff
@@ -465,6 +465,79 @@ class PodsResourceTest extends AkkaUnitTest with Mockito with JerseyTest {
       }
     }
 
+    "fail to update a pod with a persistent volume without force parameter" in {
+      implicit val podSystem = mock[PodManager]
+      val f = Fixture(configArgs = Seq("--mesos_role", "foo"))
+
+      val pathId = "/foo/mypod".toAbsolutePath
+      val existingPod = PodDefinition(id = pathId, role = "*")
```

Comment:
Actually, you're updating an existing **ephemeral** pod to a **persistent** one with a different _role_. This is a legit case and it seems it triggers the same validation. But for the completeness sake, I would also add a test updating an already **resident** pod to a different role.

Reply:
This is a great point. It's functionally ok for now.